### DISCUSSION TITLE
Personal messages

### DISCUSSION THREAD

**Discussion by cgodbolt** [community/community#147008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good morning! I have a question for anyone for who can help me. These personal emails were were on GitHub without my knowledge. Can anyone tell me how GitHub got access to this information? 
![C8B2A447-8BE6-4712-89FF-F67A2FF9F70B](https://github.com/user-attachments/assets/e7b2c7ff-cef5-40d4-b404-f5fa0b9a68cf)
![6113178E-97D1-4039-A73D-CFB92269B8CB](https://github.com/user-attachments/assets/ccca93d8-18af-45cd-8372-22e1c9f4916e)
![3E772EDB-3E2B-449D-ACA5-9117661630C2](https://github.com/user-attachments/assets/58e27c72-d5dc-4129-9fd0-b943e208ef41)

**Comment by cgodbolt** [comment]


**Comment by cgodbolt** [comment]


**Comment by mecodeatlas** [comment]
Hey @cgodbolt, thanks for reaching out!  
For this situation, I recommend checking out our documentation on Removing [sensitive data from a repository](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) and following the steps outlined there. This should help you resolve the issue. :)

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @cgodbolt!

You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Profile category. I've gone ahead and moved it for you. Good luck!

**Comment by cgodbolt** [comment]
Thank you!!
